graphistry/dep_manager.py

```diff
@@ -0,0 +1,36 @@
+import importlib
+
+class DepManager:
+    def __init__(self):
+        self.pkgs = {}
+
+    def __getattr__(self, pkg:str):
```

Comment:
Maybe we just return the imported package?

Most of the uses can check that for being not none and using, and if want version, dot into it